src/main/java/org/folio/auth/authtokenmodule/apis/RouteApi.java

```diff
@@ -150,21 +163,30 @@ private void handleSignToken(RoutingContext ctx) {
       if (isDummyTokenSigningRequest(payload)) {
         logger.debug("Signing request is for a dummy token");
 
-        token = new DummyToken(tenant, payload.getJsonArray("extra_permissions"), username);
+        var dt = new DummyToken(tenant, payload.getJsonArray("extra_permissions"), username);
+        responseObject.put("token", dt.encodeAsJWT(tokenCreator));
+        endJson(ctx, 201, responseObject.encode());
       } else {
```

Comment:
Inserting `return;` avoids the else clause and its indentation and is more easy to understand for me:

```
        endJson(ctx, 201, responseObject.encode());
        return;
      }
      logger.debug("Signing request is for an access token");

      String userId = payload.getString(USER_ID);
```

Reply:
Agree. Eliminating else is a good thing.